tests/framework/Optimizers/FD_Canyon/model.py

```diff
@@ -0,0 +1,4 @@
+import numpy as np
+
+def run(self,inp):
```

Comment:
Hey, can I ask you to add a docstring here explain what the model is supposed to model? (also the Copyright header is missing)

Reply:
agree